src/EnergyPlus/MixedAir.cc

```diff
@@ -4233,12 +4253,48 @@ namespace MixedAir {
 		// accurate result using a full mass, enthalpy and moisture balance and iteration.
 		if ( OutAirSignal > OutAirMinFrac && OutAirSignal < 1.0 && this->MixMassFlow > VerySmallMassFlow && this->ControllerType_Num == ControllerOutsideAir && ! AirLoopNightVent ) {
 			Par( 1 ) = this->MixNode;
-			Par( 2 ) = this->RetNode;
-			Par( 3 ) = this->InletNode;
+			Par( 2 ) = this->RelNode;
+			Par( 3 ) = this->OANode;
 			Par( 4 ) = this->MixMassFlow;
-			SolveRegulaFalsi( Acc, MaxIte, SolFla, OASignal, MixedAirControlTempResidual, OutAirMinFrac, 1.0, Par );
-			if ( SolFla < 0 ) {
-				OASignal = OutAirSignal;
```

Comment:
Previous solution used RegulaFalsi on static node conditions (see function below) and varying OA flow rate. This works great if only a mixer is used in the OA system but did not account for when other equipment might alter the mixed air condition at various air flow rates.